### PR TITLE
Nemo/v2/1263/convert stubs to react

### PR DESCRIPTION
**Pull request by nemodjuric**
Connects to #1263

### CODE REVIEW

File: client/app/certification/DocumentsCheckTable.jsx [hunk 1]
```diff
@@ -21,23 +24,31 @@ const DocumentsCheckTable = () => {
...
-          <td> <FoundIcon/> </td>
+          <td> { props.soc_match ? <FoundIcon/> : <NotFoundIcon/> } </td>
           <td> SOC </td>
-          <td> 03/19/2007 </td>
-          <td> 03/19/2007 </td>
+          <td> {props.soc_date} </td>
+          <td> {props.soc_date} </td>
         </tr>
+        { props.ssoc_dates.map((ssoc_date, index) =>
+          <tr key={index}>
```
Comment (amprokop) [inline client/app/certification/DocumentsCheckTable.jsx]: let's match the old version as often as we can! should we change this `key={index}` to `id="SSOC #{i+1}"`

File: app/controllers/certifications_controller.rb [hunk 2]
```diff
@@ -10,9 +10,8 @@ def new
     status = certification.start!
     @form8 = certification.form8
 
-    if verify_certification_v2_access && status == :started
+    if verify_certification_v2_access
       render "v2", layout: "application"
-      return
```
Comment (amprokop) [inline app/controllers/certifications_controller.rb]: why did you remove this `return`?

File: client/app/certification/Certification.jsx [hunk 3]
```diff
@@ -46,7 +48,13 @@ const Certification = () => {
         when we finish implementing the rest of certification v2,
         port those over here */}
       <Route path="/certifications/:vacols_id/check_documents"
-        component={DocumentsCheckSuccess}/>
+        component={DocumentsCheck}/>
+      <Route path="/certifications/:vacols_id/already_certified"
+        component={AlreadyCertified}/>
+      <Route path="/certifications/:vacols_id/not_ready"
+        component={NotReady}/>
+      <Route path="/certifications/:vacols_id/mismatched_documents"
```
Comment (amprokop) [inline client/app/certification/Certification.jsx]: We shouldn't need this route, right? we can just render `DocumentsCheck` and it will figure it out.

File: client/app/certification/DocumentsCheck.jsx [hunk 4]
```diff
@@ -1,16 +1,24 @@
...
-const DocumentsCheckSuccess = ({ match }) => {
+const DocumentsCheck = ({ match }) => {
   return <div>
     <div className="cf-app-segment cf-app-segment--alt">
       <h2>Check Documents</h2>
-      <DocumentsMatchingBox/>
-      <DocumentsCheckTable/>
+      { match ? <DocumentsMatchingBox/> : <DocumentsNotMatchingBox/> }
+      <DocumentsCheckTable
+      form9_match = { true }
```
Comment (amprokop) [inline client/app/certification/DocumentsCheck.jsx]: Can you make the indentation and whitespace match our other React code?


for example:
```
      <RadioField id={`SSOC #{i+1}`}
        name={hearingChangeQuestion}
        required={true}
        options={hearingChangeAnswers}/>
```

File: app/controllers/certifications_controller.rb (hunk 2 above)
Comment (amprokop) [inline app/controllers/certifications_controller.rb]: oh i see — it's not necessary anymore

File: client/app/certification/Certification.jsx (hunk 3 above)
Comment (amprokop) [inline client/app/certification/Certification.jsx]: for now, we can leave it as is though.

File: client/app/certification/DocumentsCheckTable.jsx [hunk 5]
```diff
@@ -21,23 +24,31 @@ const DocumentsCheckTable = () => {
...
+            <td><FoundIcon/></td>
+            <td> SSOC { index+1 }</td>
+            <td> { ssoc_date} </td>
```
Comment (amprokop) [inline client/app/certification/DocumentsCheckTable.jsx]: can you do a pass in this file and make the whitespace consistent? i think we usually prefer no spaces before/after curly braces `{thing}` not `{ thing }`

File: client/app/certification/NotReady.jsx [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+// Not used yet. TODO: use it.
```
Comment (amprokop) [inline client/app/certification/NotReady.jsx]: can probably remove this comment

Comment (amprokop) [inline client/app/certification/NotReady.jsx]: can remove this TODO i think.

File: client/app/certification/DocumentsCheckTable.jsx (hunk 1 above)
Comment (nemodjuric) [inline client/app/certification/DocumentsCheckTable.jsx]: I will add id={"ssoc-"+(index+1)+"-match}" to match the old version. However, to my understanding, keys should be given to the elements inside the array so React can keep track of changes/updates.

File: client/app/certification/Certification.jsx (hunk 3 above)
Comment (nemodjuric) [inline client/app/certification/Certification.jsx]: Correct. DocumentCheckTable covers both cases.
